docs/topics/gradle/gradle-configure-project.md

```diff
@@ -81,7 +81,8 @@ methods and properties, and supports all the current Gradle features.
 By default, the Kotlin Gradle plugin stores persistent project-specific data at the root of the project,
 in the `.kotlin` directory.
 
-> You may want to add the `.kotlin` directory to your project's `.gitignore` file.
+> Do not commit the `.kotlin` directory to version control.
+> For example, if you are using Git, add `.kotlin` to your project's `.gitignore` file.
```

Comment:
Is this only an example of what they can do, or is it the recommended practice? If it's the recommended practice, how about: 
"Instead, if you are using Git, add `.kotlin` to your project's `.gitignore` file."

Reply:
Is there something else they can do if they're not using Git?

Reply:
good point - it's an example - there could be other version control systems, but GIt is by far the most popular.